core/services/pom.xml

```diff
@@ -11,6 +11,7 @@
   <description>eXo CMS Service</description>
   <properties>
     <exo.test.coverage.ratio>0.34</exo.test.coverage.ratio>
+    <org.lombok.version>1.18.2</org.lombok.version>
```

Comment:
please move version declaration to https://github.com/Meeds-io/maven-depmgt-pom
we will need to apply this change on other projects but not in the context of current task.

Reply:
okay

Reply:
https://github.com/Meeds-io/maven-depmgt-pom/pull/34